hummingbot/connector/exchange/bybit/bybit_api_order_book_data_source.py

```diff
@@ -199,14 +184,15 @@ async def _process_ws_messages(self, ws: WSAssistant):
             data = ws_response.data
             if data.get("msg") == "Success":
                 continue
-            event_type = data.get("topic")
-            if event_type == CONSTANTS.DIFF_EVENT_TYPE:
-                if data.get("f"):
-                    self._message_queue[CONSTANTS.SNAPSHOT_EVENT_TYPE].put_nowait(data)
-                else:
-                    self._message_queue[CONSTANTS.DIFF_EVENT_TYPE].put_nowait(data)
-            elif event_type == CONSTANTS.TRADE_EVENT_TYPE:
-                self._message_queue[CONSTANTS.TRADE_EVENT_TYPE].put_nowait(data)
+
+            try:
+                data_type = data.get("type")
+                if data_type == "delta":
```

Comment:
can use ORDERBOOK_DIFF_EVENT_TYPE constant instead of "delta" string, which you added?
Also based on the api document, seems like data_type can be "snapshot", which requires "you will have to reset your local orderbook."
https://bybit-exchange.github.io/docs/v5/websocket/public/orderbook

Seems like snapshot event is not covered in the new code.

Reply:
The initial snap shot is loaded. If it was not, the order book would not be initialised by my understanding. I will re read the documentation in regards to that aspect.

Reply:
Understood, obviously, you are correct. I was only processing snapshots via the rest api, I will fix this. Bybit only has two types of packet, snapshot and delta. So I will research how to add the snap shot here and what triggers a web sockets snap shot as I have yet to see one.

Reply:
I have posted a rather simple fix and well it was a no so intelligent oversight. I will test to see if it is enough. Though I have noticed snap shots via the web sockets are very rare. I most probably will have to wait an hour as the first one is done via rest.